keyboards/plut0nium/0x3E/keymaps/default/keymap.c

```diff
@@ -0,0 +1,131 @@
+ */
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _QWERTY,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_QWERTY] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,              KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC, 
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,              KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSLS,  
+        MO(_FN), KC_A,    KC_S,    KC_D,    KC_F,    KC_G,              KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_ENT,  
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_PGUP,  KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, 
+        KC_LCTL, KC_LGUI, KC_MENU, KC_LALT, KC_SPC,  KC_SPC,  KC_PGDN,  KC_SPC,  KC_SPC,  KC_RALT, KC_LBRC, KC_RBRC, KC_RCTL  
+    ),
+    [_FN] = LAYOUT(
+        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,             KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_DEL,  
+        RESET,   _______, KC_UP,   _______, _______, _______,           _______, _______, _______, KC_F11,  KC_F12,  _______, 
+        _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,           _______, _______, _______, _______, _______, _______,
+        _______, RGB_HUI, RGB_SAI, RGB_VAI, RGB_MOD, _______, KC_END,   _______, _______, BL_INC,  KC_MPLY, KC_VOLU, KC_MUTE,
+        _______, RGB_HUD, RGB_SAD, RGB_VAD, RGB_TOG, RGB_TOG, KC_HOME,  BL_TOGG, BL_TOGG, BL_DEC,  KC_MPRV, KC_VOLD, KC_MNXT
+    )
+};
+
+#ifdef OLED_DRIVER_ENABLE
+static void render_logo(void) {
+    static const char PROGMEM my_logo[] = {
+        // '0x3E_logo_32x16', 32x16px
+        0xff, 0x0f, 0x07, 0xf3, 0x73, 0xb3, 0x07, 0x0f, 0xff, 0x3f, 0x3f, 0xff, 0xff, 0x3f, 0x3f, 0xff, 
+        0xe7, 0xe3, 0x33, 0x33, 0x03, 0x87, 0xff, 0xff, 0x03, 0x03, 0x33, 0x33, 0x33, 0xf3, 0xff, 0xff, 
+        0x7f, 0x78, 0x70, 0x66, 0x67, 0x67, 0x70, 0x78, 0x7f, 0x67, 0x62, 0x78, 0x78, 0x62, 0x67, 0x7f, 
+        0x73, 0x63, 0x67, 0x67, 0x60, 0x70, 0x7f, 0x7f, 0x60, 0x60, 0x67, 0x67, 0x67, 0x67, 0x7f, 0x7f
+    };  
+    oled_write_raw_P(my_logo, sizeof(my_logo));
+}
+
+#ifdef RGBLIGHT_ENABLE
+extern rgblight_config_t rgblight_config;
+
+void render_rgb_status(void) {
+    oled_write_ln_P(PSTR("RGB"), false);
+    if (!rgblight_config.enable) {
+        oled_write_P(PSTR("  off\n\n\n"), false);
+    }
+    else {
+        static char led_buf[24] = {0};
+        snprintf(led_buf, sizeof(led_buf), "M:%3dH:%3dS:%3dV:%3d",
+            (uint8_t)rgblight_config.mode,
+            (uint8_t)rgblight_config.hue,
+            (uint8_t)rgblight_config.sat,
+            (uint8_t)rgblight_config.val);
+        oled_write(led_buf, false);
+    }
+}
+#endif
+
+extern backlight_config_t backlight_config;
+
+void render_backlight_status(void) {
+    oled_write_ln_P(PSTR("BKL"), false);
+    if (!backlight_config.enable) {
```

Comment:
Here also, previous suggestion breaks this
```suggestion
    if (!is_backlight_enabled()) {
```